api/v2/v2.go

```diff
@@ -194,35 +199,45 @@ type VerifyBatch struct {
 	Timestamps []int64  `json:"timestamps"`
 }
 
-// VerifyBatchReply is returned by the server with the status results for the requested
-// digests and timestamps.
+// VerifyBatchReply is returned by the server with the status results for the
+// requested digests and timestamps.
 type VerifyBatchReply struct {
 	ID         string            `json:"id"`
 	Digests    []VerifyDigest    `json:"digests"`
 	Timestamps []VerifyTimestamp `json:"timestamps"`
 }
 
-// ChainInformation is returned by the server on a verify digest request. It contains
-// the merkle path of that digest.
+// ChainInformation is returned by the server on a verify digest request.
+// It contains the merkle path of that digest.
 type ChainInformation struct {
 	ChainTimestamp int64         `json:"chaintimestamp"`
 	Transaction    string        `json:"transaction"`
 	MerkleRoot     string        `json:"merkleroot"`
 	MerklePath     merkle.Branch `json:"merklepath"`
 }
 
-// CollectionInformation is returned by the server on a verify timestamp request.
-// It contains all digests grouped on the collection of the requested block timestamp.
+// CollectionInformation is returned by the server on a verify timestamp
+// request. It contains all digests grouped on the collection of the
+// requested block timestamp.
 type CollectionInformation struct {
 	ChainTimestamp int64    `json:"chaintimestamp"`
 	Transaction    string   `json:"transaction"`
 	MerkleRoot     string   `json:"merkleroot"`
 	Digests        []string `json:"digests"`
 }
 
-// WalletBalanceReply returns balance information of the decred wallet.
+// WalletBalanceReply is returned by server on a balance information of the
+// decred wallet.
 type WalletBalanceReply struct {
 	Total       int64 `json:"total"`
 	Spendable   int64 `json:"spendable"`
 	Unconfirmed int64 `json:"unconfirmed"`
 }
+
+// LastAnchorReply is returned by server on a last anchor info request
```

Comment:
What is considered an anchor?  When the tx gets sent, once it has 6 confirmations, or some other criteria?